cub/cub/device/device_reduce.cuh

```diff
@@ -1035,7 +1035,7 @@ struct DeviceReduce
   //!    struct CustomMin
   //!    {
   //!        template <typename T>
-  //!        CUB_RUNTIME_FUNCTION __forceinline__
+  //!        __host__ __device__ __forceinline__
```

Comment:
Question: Why are you not using `_CCCL_HOST_DEVICE`?

Reply:
To my understanding, `_CCCL_HOST_DEVICE` is for internal usage and something we wouldn't want to leak into the examples.

Reply:
Ah, I was just looking at the file names and thought you were editing CUB internal code. You are in fact editing documentation though. All good, sorry for the confusion!